pir/preload.bzl

```diff
@@ -30,9 +29,9 @@ def pir_preload():
         )
 
     if "rules_foreign_cc" not in native.existing_rules():
-        git_repository(
+        http_archive(
```

Comment:
We had http_archive before, and it broke our build system.

More specifically, this commit https://github.com/bazelbuild/rules_foreign_cc/commit/d838bc55067fc473fa67ec619be1478e45b96e2d

was creating some issues, and there was no way to avoid it using http_archive. 

Maybe they fixed the issue on master, but that's why we switched to git_repository here.

Reply:
I see. Since it appears to be working now, shall we merge?

Reply:
yes, LGTM